third_party/dml/ci/test/run_tests.py

```diff
@@ -69,7 +72,22 @@ def main():
         for flag in json_test_group["flags"]:
           command_line.append(flag)
 
-      conda_env.run(command_line)
+      if not is_python_test and args.tensorflow_wheel is not None:
+        # .whl files are simple zip files, so we can extract DirectML.dll from
+        # it
+        with TemporaryDirectory() as temp_dir:
+          with ZipFile(args.tensorflow_wheel, 'r') as zip_wheel:
+            zip_wheel.extractall(temp_dir)
```

Comment:
Nit: the wheels can be slow to extract, so it might be better to just extract the solib_directml folder. Though really a drop in the bucket for total test time... feel free to WontFix.